.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 18.x, 20.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
@enov Ah, I see I did the same thing here. Sorry for correcting that, I had pushed this in a second commit and I think I had intended to do that as a separate PR. Sorry for asking you fix that just to re-do it here and have me done it in the same way 🤦

Reply:
No worries :smile: